.github/workflows/security-tests.yml

```diff
@@ -15,20 +16,18 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Make .env file
-        uses: SpicyPizza/create-envfile@v1.3
-        with:
-          envkey_API_KEY: "${{ secrets.TRAMS_API_KEY }}"
-          envkey_URL: "${{ secrets.TRAMS_API_ENDPOINT }}"
-          envkey_HTTP_PROXY: "http://zap:8080"
-          envkey_NO_PROXY: "zap,*.gvt1.com"
-          envkey_ZAP_API_KEY: "" # TODO add value to github secrets
-          directory: CypressTests
-          fail_on_empty: true
+      - name: Prepare environment
+        run: |
+          echo "{API_KEY}={${{ secrets.TRAMS_API_KEY }}}" >> $GITHUB_ENV
```

Comment:
```bash
echo "API_KEY=${{ secrets.TRAMS_API_KEY }}" >> $GITHUB_ENV
```
should be sufficient formatting here.

example from docs:
```bash
echo "action_state=yellow" >> $GITHUB_ENV
```

Reply:
Another option is to add them before the docker compose (line 29).
like this -
`
env:
     HTTP_PROXY : http://zap:8080
     ZAP_API_KEY : ${ secrets.ZAP_API_KEY }
     etc etc
`